src/ui/Header.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import SettingsCard from './SettingsButton';
+import {
+  Flex,
+  Link,
+  HStack,
+  Button as ChakraButton,
+  Text,
+} from '@chakra-ui/react';
+import { Icon, IconNames } from '@nypl/design-system-react-components';
+import { WebpubManifest, ReaderState, Navigator } from '../types';
+
+export type HeaderProps = {
+  logo?: any;
```

Comment:
I wouldn't make this separate from the "Return to Digital Research Books" text - just let anyone pass whatever they want.  Not sure what we'd call this variable but if we put a comment saying it's the thing on the top left I'm sure people will understand.